### ISSUE TITLE
Tina Media Not Configured

### ISSUE THREAD

**Issue by newash** [tinacms/tinacms#4043]
### Describe the issue that you're seeing. Any Loom videos or screenshots usually help a lot!

While fiddling with my setup and waiting for my other issue #3569 to magically resolve itself,
I updated my Tina packages and my setup got worse.

My setup is as in my  https://github.com/tinacms/tinacms/issues/3569#issuecomment-1577396506 there:
```js
  media: {
    tina: {
      publicFolder: "content",
      mediaRoot: "assets",
    },
  },
```

This is what I'm getting on my website's Media Manager page:
![2023-06-26_23h55_45](https://github.com/tinacms/tinacms/assets/4682432/21247a0f-98ee-4d62-8217-6dc1b6bcc96e)

And this is that I get when I navigate to Tina Cloud with that link there:
![2023-06-26_23h55_53](https://github.com/tinacms/tinacms/assets/4682432/bc3171cc-ef91-4245-bf04-5041afd4e704)

Those folders set above exist in the repo.

Why am I getting these messages?

### Reproduction

https://feature-tina--eztettem.netlify.app/admin/

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.2 LTS 22.04.2 LTS (Jammy Jellyfish)
    CPU: (8) x64 AMD Ryzen 7 PRO 3700U w/ Radeon Vega Mobile Gfx
    Memory: 12.30 GB / 14.61 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.16.0 - /usr/bin/node
    Yarn: 1.22.19 - ~/Work/eztettem/node_modules/.bin/yarn
    npm: 9.5.1 - /usr/bin/npm
  npmPackages:
    @tinacms/cli: ^1.5.20 => 1.5.20 
    tinacms: ^1.5.11 => 1.5.11
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/tinacms/tinacms/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://tina.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/tinacms/tinacms/issues) that reports the same bug.

**Comment by kldavis4** [comment]
@newash this issue can occur when you don't have Tina (`tina/tina-lock.json + `tina/config.[ext]`) in your default branch.

**Comment by newash** [comment]
aargh!!!

I mean: thanks for the life-saving info, I'd have never figured that out myself. :)

It's only gonna be a pain in the a*** since Tina is only in my dev branch, I cannot switch to it until it fully works.

**Comment by kldavis4** [comment]
Yeah, this is something we hope to improve. If you just copy over those two files to your default branch, you can hack it into working without fully installing tina on that branch.

**Comment by jamespohalloran** [comment]
I added a blurb in the docs to hopefully make this smoother in the future: https://github.com/tinacms/tina.io/pull/1598
In the medium term, I don't think it would be a huge lift for us to make the media branch configurable to get things working before Tina's been merged to the default branch.

**Comment by jmarks-joshua** [comment]
I'm getting this same issue even though I do have both tina/tina-lock.json and tina/config.tsx in my default branch. Below is my config. Is it maybe because my media root is an empty string? 

```import { defineConfig } from "tinacms";
import { SimpleContentTemplate } from "./templates";

// Your hosting provider likely exposes this as an environment variable
const branch = process.env.HEAD || process.env.VERCEL_GIT_COMMIT_REF || "tina";

export default defineConfig({
  branch,
  clientId: "56163663-3970-4413-bb28-fc97e8f375f0", // Get this from tina.io
  token: "1c9124fb264ca5504ad6bda0ca7d1a047ba03b9e", // Get this from tina.io

  build: {
    outputFolder: "admin",
    publicFolder: "public",
  },
  media: {
    tina: {
      mediaRoot: "",
      publicFolder: "public",
    },
  },
  schema: {
    collections: [
      {
        name: "content",
        label: "Fixed content",
        path: "content/content",
        format: "mdx",
        templates: [SimpleContentTemplate],
        ui: {
          router: ({ document }) => `/content/${document._sys.filename}`,
        },
      },
    ],
  },
});
```

**Comment by jmarks-joshua** [comment]
There's nothing to screen record I just see this 

![image](https://github.com/tinacms/tinacms/assets/2428096/ed51870c-9089-46e4-8e2c-4743579dc53f)

**Comment by kldavis4** [comment]
Change the media branch to `main`

**Comment by jmarks-joshua** [comment]
tina is the only branch I have configured main, doesn't appear in the dropdown. I am currently exclusively using the tina branch. If you think that's the cause of all this I'll sort that, but seems odd that it would be that.

**Comment by kldavis4** [comment]
Does it do anything when you click on the drop down? There might be a bug in the UI. cc @jamespohalloran

**Comment by jmarks-joshua** [comment]
Yeah just gives me a dropdown with the single option of tina

![Screenshot (2)](https://github.com/tinacms/tinacms/assets/2428096/d7976693-a10b-48b4-a837-bec100815b64)

**Comment by kldavis4** [comment]
Please try the workaround above: https://github.com/tinacms/tinacms/issues/4043#issuecomment-1609508109

**Comment by jmarks-joshua** [comment]
I don't understand all of my branches have those files committed.

**Comment by jamespohalloran** [comment]
I'd be curious to see what the network request looks like for when you load the app in the dashboard.

If you could go from the project list view with the network-tab open, navigate to the project, then take a screenshot of this request. It would be helpful to know if we're fetching your updated media metadata for your given `tina` branch:

<img width="1466" alt="Screenshot 2023-09-12 at 11 56 40 AM" src="https://github.com/tinacms/tinacms/assets/3323181/633fc458-3f3f-46c4-9d1f-9bbd8f979f96">

**Comment by kldavis4** [comment]
@jmarks-joshua if you go to the configuration tab in the project, you will see the list of branches that tina knows about, and that is currently only the `tina` branch.

**Comment by kldavis4** [comment]
> Yes this is correct but both my main and tina branches are identical. If this issue is because Tina doesn't have access to my main branch for some reason how do I give it access? I can't find much about this in the docs.

Does `tina/tina-lock.json` and `tina/config[.*]` exist in your `main` branch?

**Comment by jmarks-joshua** [comment]
Yes both exist they are identical branches. My main branch I just did a 
`git reset --hard b9e299c1eb5194befb76e9913f763c55c0587d6e`
which is the SHA of the latest tina branch commit.

**Comment by kldavis4** [comment]
And you are sure `main` exists in github (and not just locally)? Push `main` up and it should find it if it is identical.

**Comment by jmarks-joshua** [comment]
It's all pushed up I promise. 

![image](https://github.com/tinacms/tinacms/assets/2428096/9cd9f300-256d-4573-b81f-823034ade4db)

Is there anything I might have done at setup that might have restricted access to only one branch? I've had a look and I can't see anywhere that might have happened.

**Comment by jamespohalloran** [comment]
In that screenshot, it looks like your tina-lock.json was updated last 2 months ago. If you run yarn install, yarn dev (or tinacms dev), does it get updated locally? Each time you update the schema/config, that file should be updated & re-pushed to git.

**Comment by jmarks-joshua** [comment]
Yeah that all works, it was just a project that I finished up 2 months ago waiting for clients to look at and we're finally back on the project. 

None the less seems to have all been resolved by just making a white space change to my tina-lock.json file which triggered the branch to pop up. I've moved everything over to use main for now as seems like that's what's expected on Tina's end. 

Thanks for all your help.
